packages/@ama-sdk/core/package.json

```diff
@@ -43,19 +42,14 @@
     "tslib": "^2.4.1",
     "uuid": "^8.3.2"
   },
-  "peerDependenciesMeta": {
-    "@angular-eslint/template-parser": {
-      "optional": true
-    },
-    "@angular/compiler": {
-      "optional": true
-    }
-  },
   "peerDependencies": {
     "isomorphic-fetch": "^2.2.1"
```

Comment:
shouldn't `"@angular-devkit/schematics": "~15.0.4",` be part of the peerDeps as well? isn't `yarn in-source-dep-check` flagging this as a warning?

Reply:
@angular-devkit/schematics is use only for an import type so devDep is enough I think